docs/ota-client-guide/modules/ROOT/pages/evaluation-to-prod.adoc

```diff
@@ -8,19 +8,26 @@ We recommend that you link:https://docs.ota.here.com/ota-client/latest/{docname}
 endif::[]
 
 
-The procedures to deploy OTA Connect in production are a little more complex than the basic xref:getstarted::index.adoc[Get Started]  procedures.
+The procedures to deploy OTA Connect in a realistic production scenario are a little more complex than the basic xref:getstarted::index.adoc[Get Started] procedures. However, you don't need to commit the resources for a full production roll-out right at the beginning of the process.
 
-That's why it's better work in phases when setting up your OTA Connect in your organization. We recommend that you work in three main phases:
+That's why it's better to work in phases when setting up OTA Connect in your organization. We recommend that you work in three main phases:
 
 . __Evaluate OTA Connect__
-. __Build your own OTA-enabled solution__
-. __Deploy your own OTA-enabled solution__
+. __Build a realistic OTA-enabled pilot project__
+. __Deploy a fully production-ready OTA solution__
```

Comment:
why do we talk about _project_ in one phase and then _solution_ in the next phase ?

Reply:
Because you can't really have a "pilot solution". The pilot phase is about trying stuff out and making sure it works; the production phase it about solving your problem.